app/forms/idv/consent_form.rb

```diff
@@ -2,15 +2,23 @@ module Idv
   class ConsentForm
     include ActiveModel::Model
 
-    validates :idv_consent_given?,
+    validates :ial2_consent_given,
```

Comment:
I bet we don't need to define a separate `ial2_consent_given`, we could just reuse the `idv_consent_given?` accessor right?

Reply:
Yeah I think this got caught up in a separate effort to rename that field, I've updated in d34a937